src/app/modules/map/components/map-center-info/map-center-info.component.html

```diff
@@ -45,8 +45,8 @@
       </ng-container>
     </ion-row>
 
-    <ion-row name="weather-link-row" (click)="$event.stopPropagation(); loadYr(mapCenter.lat, mapCenter.lng)">
-      <div class="weather-link">
+    <ion-row name="weather-link-row">
+      <div role="link" class="weather-link" (click)="$event.stopPropagation(); loadYr(mapCenter.lat, mapCenter.lng)">
```

Comment:
For meg ser det ut som posisjonen er tilgjengelig for denne funksjonen siden mapCenter er en property i klassen, så lurer på om du trenger å sende dennee inn fra html-malen?

Reply:
Jeg bare flyttet denne funksjonen fra ion-row. Var ikke meg som implementerte den. Vil si at det ikke er scopet til denne oppgaven :P